app/controllers/my_module_tags_controller.rb

```diff
@@ -63,13 +63,10 @@ def create
               my_module: my_module.id,
               tag: @mt.tag.id
             })
-
-    respond_to do |format|
-      format.json do
-        redirect_to my_module_tags_edit_path(format: :json), turbolinks: false,
-                    status: 303
-      end
-    end
+    # rubocop:disable Rails/HttpStatus
+    redirect_to my_module_tags_edit_path(format: :json), turbolinks: false,
+                status: 303
```

Comment:
Why just not use :see_other here?

Reply:
@rekonder Can you please help with what **:see_other** means?